CHANGELOG.md

```diff
@@ -5,7 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.2.2] - 2023-08-22
+## [1.2.3] - 2023-08-23
+
+### Fixed
+- Don't use `Ord` in `PartialOrd` implementations if using any bounds.
+
+## [1.2.2] - 2023-08-22 [YANKED]
```

Comment:
At first I thought this was supposed to be a link, and was confused where the address would be

Reply:
Yeah, just following what https://keepachangelog.com/en/1.0.0/#yanked says.